draft-ietf-teep-architecture.md

```diff
@@ -798,12 +800,12 @@ for any other TA, as discussed in {{trust-anchors-in-teep-agent}}.
 The Trust Anchor Store in a TAM consists of a list of Trust Anchors, which
 are certificates that sign various device TEE certificates.  A TAM will accept a
 device for TA management if the TEE in the device uses a TEE certificate
-that is chained to a certificate that the TAM trusts.
+that is chained to a certificate or raw public key that the TAM trusts, is contained in an allow list, 
+is not found on a block list, and/or fulfills any other policy criteria.
```

Comment:
Tiru asks: should we add a reference somewhere to an RFC that talks about the raw public key?

Reply:
Tiru asks: should (or does?) sec consid section talk about compromise of raw public key trust anchor?

Reply:
I added the term raw public key to the terminology section. The compromise of a raw public key trust anchor is not different from the compromise of a "certificate" trust anchor.